main.go

```diff
@@ -44,6 +44,9 @@ var (
 	skipForeignKeys  bool
 	sessionJSON      string
 	webapi           bool
+
+	// Flags not exposed in the documentation.
+	maxWorkers = int64(10)
```

Comment:
On a second thought, this flag is very specific to foreign keys only and thus it might make sense to put it in main.go at all ~ that would also remove the need to unnecessary pass through this extra flag throughout the codebase ..

the alternative in this case would be to define it as a variable in conversion/conversion.go such that those looking for it can find it given that this is open source

Reply:
Agreed. But instead of declaring it globally in conversion/conversion.go, I think we should declare it in cmd/cmd.go and pass it directly as a parameter to UpdateDDLForeignKey(). Otherwise we cannot control the number of workers when calling the function from various unit tests.